library/psa_crypto_driver_wrappers.h

```diff
@@ -83,6 +83,7 @@ psa_status_t psa_driver_wrapper_export_public_key(
 
 psa_status_t psa_driver_wrapper_get_key_buffer_size(
     const psa_key_attributes_t *attributes,
+    size_t key_bits,
```

Comment:
* \param[in] attributes  The key attributes.
 * \param[in] key_bits    The actual key bits used to deduce the size, given
 *                        that this could vary from the core key_bits in the
 *                        key attributes.

If this is api is invoked from  psa_import_key, there is a chance that psa_get_key_bits(attributes) might not match the actual key_bits in the data imported, from which we want to derive the key size. 
So the key_bits is what will will be used for sizing.
psa_get_key_bits(attributes)  is ignored. If one wants to invoke the API knowing fully well that psa_get_key_bits(attributes) are correct, that should be passed as key_bits.

Reply:
Please explain this in comments in the code then. The salient point seems to be that `attributes` is data provided by the caller, and therefore not trustworthy. It should also be made clear that the meaning of `psa_get_key_bits(attributes)` being 0 is that the key size must be inferred completely from the data.

It needs to be clear from the documentation what the consequences are if `psa_get_key_bits(attributes)` is invalid for the given data, and whose job it is to check.